src/comlink.ts

```diff
@@ -496,7 +501,15 @@ function createProxy<T>(
         }).then(fromWireValue);
         return r.then.bind(r);
       }
-      return createProxy(ep, pendingListeners, [...path, prop]);
+
+      const cachedProxy = cachedProxies.get(prop);
+      if (cachedProxy) {
+        return cachedProxy;
+      }
+
+      const subProxy = createProxy(ep, pendingListeners, [...path, prop]);
```

Comment:
nit: I'd probably call this variable `propProxy` since it is a proxy for the prop - which is a function call in this case?

Reply:
> which is a function call in this case?

No, not necessarily. The prop can also just be a simple primitive (string, number, ...) or another object